api/v1alpha1/tls_types.go

```diff
@@ -115,6 +115,10 @@ type ClientValidationContext struct {
 	// +optional
 	Optional bool `json:"optional,omitempty"`
 
+	// Configure Envoy proxy how to handle the x-forwarded-client-cert (XFCC) HTTP header.
```

Comment:
when we discussed this in the last community meeting, @zhaohuabing raised a good point that this field doesnt always tie into `ClientValidation`, since `Mode` setting does not require `ClientValidation` to be set.
we could probably call this field `XForwardedClientCert ` and define it under `Headers`.
* an additional CEL check could be added later to make sure that `XForwardedClientCert.Set` can only be set when `TLS.ClientValidation != nil`